src/Carbon/Lang/my.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Carbon package.
+ *
+ * (c) Brian Nesbitt <brian@nesbot.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+return array(
+    'year' => ':count နှစ်|:count နှစ်',
+    'y' => ':count နှစ်|:count နှစ်',
+    'month' => ':count လ|:count လ',
+    'm' => ':count လ|:count လ',
+    'week' => ':count ပတ်|:count ပတ်',
+    'w' => ':count ပတ်|:count ပတ်',
+    'day' => ':count ရက်|:count ရက်',
+    'd' => ':count ရက်|:count ရက်',
+    'hour' => ':count နာရီ|:count နာရီ',
+    'h' => ':count နာရီ|:count နာရီ',
+    'minute' => ':count မိနစ်|:count မိနစ်',
+    'min' => ':count မိနစ်|:count မိနစ်',
+    'second' => ':count စက္ကန့်|:count စက္ကန့်',
+    's' => ':count စက္ကန့်|:count စက္ကန့်',
+    'ago' => 'လွန်ခဲ့တဲ့ :time က',
+    'from_now' => 'ယခုမှစ၍နောက် :time အကြာ',
+    'after' => ':time ကြာပြီးနောက်',
+    'before' => ':time မတိုင်ခင်က',
+    'diff_now' => 'အခုလေးတင်',
+    'diff_yesterday' => 'မနေ့က',
+    'diff_tomorrow' => 'မနက်ဖြန်',
+    'diff_before_yesterday' => 'တမြန်နေ့က',
+    'diff_after_tomorrow' => 'တဘက်ခါ',
+    'period_recurrences' => 'once|:count ကြိမ်',
```

Comment:
If there is no word for "once" (1 time) in Myanmar language, please just put `'period_recurrences' => ':count ကြိမ်',` or the singular form of `ကြိမ်` if it exists.

Reply:
Ohh, sorry. My bad. I will fix it.